rmgpy/tools/sensitivity.py

```diff
@@ -77,7 +76,7 @@ def plotSensitivity(outputDirectory, reactionSystemIndex, sensitiveSpeciesList,
 
 
 
-def simulate(rmg):
+def simulate(rmg, diffusionLimited=True):
```

Comment:
Was this bit added in by accident?  I think this stuff belongs with the last commit.

Reply:
Indeed it was. Not sure how that got in there.